.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,18 +4,24 @@ about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing li
 labels: "kind/bug/possible"
 
 ---
-
+<!--
 Please post all questions and issues on [janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)
```

Comment:
I think we have three channels now: Mailing list, gitter, and GitHub discussions.